docs/index.md

```diff
@@ -111,11 +111,13 @@ Lieer can be configured using `gmi set`. Use without any options to get a list o
 
 **`historyId`** is the latest synced GMail revision. Anything since this ID will be fetched on the next [`gmi pull`](#pull) (partial).
 
+**`age`** is the maximum age of email. This is an partial implementation of [the `newer_than` search operator](https://support.google.com/mail/answer/7190?hl=en). Anything email newer than this age (unit: month) will be fetched on the next [`gmi pull`](#pull). If you change this parameter to an older age, you need to do a full pull.
+
```

Comment:
`Anything` -> `Any`. Will this cause emails to be deleted (locally) when they go out of the newer_than window? This should probably refer to the section about changing some crucial config settings.

Reply:
Thank you for reminding me of this typo. Based on my understanding of the lieer code and the Gmail API docs, setting age to a smaller number will cause emails to be deleted which has been confirmed by a manual test.